src/Console/ConfigurationResolver.php

```diff
@@ -253,7 +255,25 @@ public function getConfigFile()
     public function getDiffer()
     {
         if (null === $this->differ) {
-            $this->differ = false === $this->options['diff'] ? new NullDiffer() : new SebastianBergmannDiffer();
+            switch ($this->options['diff']) {
+                case false:
```

Comment:
Yes, since we allow passing an `array` it could be even stranger like Resources.
 However, since we pass directly values as parsed by InputOption it should not be possible to be `null`. Updating to support sounds fair to me though, I'll make the change.